static/css/event.less

```diff
@@ -0,0 +1,102 @@
+/* ==========================================================================
+   cfgov-refresh
+   Event post detail page
+   ========================================================================== */
+
+.event-status {
```

Comment:
Yep, seeing that now as well. Will remove them in the cleanup I'm currently doing.